metron-analytics/metron-profiler-client/pom.xml

```diff
@@ -77,10 +77,15 @@
             <version>${global_hadoop_version}</version>
         </dependency>
         <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.3</version>
+        </dependency>
+        <dependency>
```

Comment:
Have you considered just depending on metron-hbase which will bring in the hbase-client?  I think it should take care of excluding the disruptor business and everything.